console/views.py

```diff
@@ -20,9 +24,13 @@ def home():
     return "stuff"
 
 
-def send_data(url, data):
+def send_data(url, data=None, request_type="POST"):
     try:
-        r = requests.post(url, data)
+        if request_type == "POST":
+            logger.info("POSTING DATA")
```

Comment:
Should add some more info to the log message here, if there's nothing relevant to log i think we can just remove it